### PR TITLE
Siva Reddy | Upgrading openmrs version modules

### PR DESCRIPTION
**Pull request by sivareddyp**
- Upgrading OpenMRS version to 2.5.12
- bahmnicore 1.2.0-SNAPSHOT (includes SNOMED milestone 1 changes)
- Also, updated modules to latest version

### CODE REVIEW

File: distro/pom.xml [hunk 1]
```diff
@@ -15,46 +15,46 @@
...
-        <openConceptLabVersion>1.3.0</openConceptLabVersion>
-        <initializerModuleVersion>2.4.0</initializerModuleVersion>
-        <bahmniCommonsVersion>1.0.0</bahmniCommonsVersion>
-        <teleconsultationVersion>1.1.0</teleconsultationVersion>
-        <communicationVersion>1.1.0</communicationVersion>
+        <webServicesRestVersion>2.40.0</webServicesRestVersion>
+        <reportingRestVersion>1.14.0</reportingRestVersion>
+        <bahmniIEOmodVersion>1.4.0-SNAPSHOT</bahmniIEOmodVersion>
+        <appointmentsVersion>1.8.0-SNAPSHOT</appointmentsVersion>
+        <pacsQueryVersion>1.5.0-SNAPSHOT</pacsQueryVersion>
```
Comment (angshu) [inline distro/pom.xml:50]: has anything changed over here? We don't have to up the dependent module version unless we are consuming them.

File: distro/pom.xml [hunk 2]
```diff
@@ -15,46 +15,46 @@
...
+        <fhir2ModuleVersion>1.10.0-SNAPSHOT</fhir2ModuleVersion>
+        <fhir2ExtensionModuleVersion>1.3.0-SNAPSHOT</fhir2ExtensionModuleVersion>
+        <openConceptLabVersion>2.1.0</openConceptLabVersion>
+        <initializerModuleVersion>2.5.2</initializerModuleVersion>
+        <bahmniCommonsVersion>1.1.0-SNAPSHOT</bahmniCommonsVersion>
+        <teleconsultationVersion>1.2.0-SNAPSHOT</teleconsultationVersion>
```
Comment (angshu) [inline distro/pom.xml:56]: same question here. Are there feature from teleconsultation 1.2.0-SNAPSHOT that we are using?

File: distro/pom.xml [hunk 3]
```diff
@@ -15,46 +15,46 @@
...
-        <htmlwidgetsVersion>1.10.0</htmlwidgetsVersion>
+        <htmlwidgetsVersion>1.11.0</htmlwidgetsVersion>
         <idgenModuleVersion>4.7.0</idgenModuleVersion>
         <idgenWebServicesModuleVersion>1.4.0</idgenWebServicesModuleVersion>
         <legacyUiOmodVersion>1.13.0</legacyUiOmodVersion>
-        <mailappenderVersion>0.94.3</mailappenderVersion>
-        <metadataMappingVersion>1.5.0</metadataMappingVersion>
-        <metadataSharingVersion>1.8.0</metadataSharingVersion>
-        <openMRSVersion>2.5.10</openMRSVersion>
+        <mailappenderVersion>0.95.0-SNAPSHOT</mailappenderVersion>
```
Comment (angshu) [inline distro/pom.xml:34]: are we using this version?

File: distro/pom.xml (hunk 1 above)
Comment (sivareddyp) [reply distro/pom.xml]: reverted this change.

File: distro/pom.xml (hunk 2 above)
Comment (sivareddyp) [reply distro/pom.xml]: reverted this change.

**Pull request closed without merging**